include/curl/curl.h

```diff
@@ -1575,7 +1575,7 @@ typedef enum {
   /* set the bitmask for the protocols that libcurl is allowed to follow to,
      as a subset of the CURLOPT_PROTOCOLS ones. That means the protocol needs
      to be set in both bitmasks to be allowed to get redirected to. Defaults
-     to all protocols except FILE and SCP. */
+     to HTTP, HTTPS, FTP and FTPS. */
```

Comment:
I say just remove those details from this comment. It's a bit over the top to have this written everywhere methinks.